firefox-ios/Client/Frontend/Browser/Search/SearchViewModel.swift

```diff
@@ -136,7 +138,9 @@ class SearchViewModel: FeatureFlaggable, LoaderListener {
         hasHistorySuggestions
     }
 
+    /// Does not show when search term in url is empty (aka zero search state).
     var hasFirefoxSuggestions: Bool {
+        guard !searchQuery.isEmpty else { return false }
```

Comment:
would love a better way to do the `searchQuery.isEmpty` check in one place, but this would require a larger refactor in the search area. So I checked it in any places where I see the suggestions populating in the zero state when we should only see trending searches.